src/connection-stage-actions.ts

```diff
@@ -79,11 +84,8 @@ export class ConnectionStageActions {
       }
       case ConnectionFlowType.RadioBridge: {
         newStage = {
-          ...this.stage,
-          connType: "radio",
-          flowStep: ConnectionFlowStep.ConnectingMicrobits,
+          ...this.getConnectingStage("bluetooth"),
```

Comment:
We think this is flipped and should be radio

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/279/commits/9a18509ddeb1ac32ada920d31eae106e542e6bff